app-text/pelican/pelican-4.9.1.ebuild

```diff
@@ -56,6 +56,17 @@ EPYTEST_DESELECT=(
 
 distutils_enable_tests pytest
 
+python_prepare_all() {
+	# bug 863962
+	if use elibc_musl
```

Comment:
Let's [avoid conditional patching](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching) if we can.

It's only a handful which failed - we could `EPYTEST_DESELECT` all of those, or we could patch in something like GENTOO_SKIP_LOCALE_TESTS to `locale_available` and set that for musl. I'd probably prefer the former.

Reply:
Sure, done.